PDF/Viewer/IPDFViewer.SuperMemo.cs

```diff
@@ -206,6 +212,25 @@ protected bool CreateSMExtract()
       return ret;
     }
 
+    protected async void CreateSMExtractWithPriority()
+    {
+
+      var result = await Show.Window()
+                         .For(new Prompt<double> { Title = "Extract Priority?", Value = -1 });
```

Comment:
Could you:
- Set the default value to `Config.SMExtractPriority`
- Add this comment `// TODO: Create a better generic Prompt with: a) a description parameter, b) constraints parameters`